hierarc/Likelihood/parameter_scaling.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         """
         self._anisotropy_model = anisotropy_model
         self._evalute_ani = False
+        self.log_m2l_population_lvl = False
```

Comment:
```suggestion
        self._is_log_m2l_population_level = False
```

Reply:
The part "is" denotes that this variable is a boolean flag. The "_" at the beginning is if this class variable is not supposed to be accessed outside its own class methods.